hphp/hack/test/mro/xhp.php.exp

```diff
@@ -1,6 +1,6 @@
 Linearization for class \:x:
-[\:x]
+[\:x(PAT)]
```

Comment:
Do you know why this has changed? It's not obvious to me.

Reply:
Fixed in the last commit. Maybe the check should be flipped but `mro_passthrough_abstract_typeconst` was comparing the class kind to just Cnormal